lm-cli/lm_cli/exceptions.py

```diff
@@ -0,0 +1,96 @@
+"""
+Provide exceptions and custom handlers for the CLI.
+"""
+
+from functools import wraps
+
+import buzz
+import typer
+from loguru import logger
+from rich import traceback
+from rich.console import Console
+from rich.panel import Panel
+
+from lm_cli.config import OV_CONTACT
+from lm_cli.text_tools import dedent, unwrap
+
+
+# Enables prettified traceback printing via rich
+traceback.install()
+
+
+class LicenseManagerCliError(buzz.Buzz):
+    """
+    A generic exception base class to use in License Manager CLI
+    """
+
+
+class Abort(buzz.Buzz):
```

Comment:
A minor suggestion. I'm not sure if it'll have any impact on the error tracing, but `Abort` is a specific error for LM-CLI, so it could be derivated from the general exception `LicenseManagerCliError`.

```suggestion
class Abort(LicenseManagerCliError):
```

Reply:
Thanks, I applied it in the latest commit.